### PR TITLE
issues.js - group issues by languages and extract important labels

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
WIP, grouping issues by programming language

### CODE REVIEW

File: src/templates/issues.handlebars [hunk 1]
```diff
@@ -48,13 +36,11 @@
...
 	                 </span>
 	              </div>
-                  <div class="t-xxxs r-iblock">
-                     {{#each labels}}
-                         {{#eq (slug name) 'priorityhigh'}}
-                             <div class="r-iblock btn" style="border-color:#{{color}}; color:#{{color}};">{{name}}</div>
-                         {{/eq}}
-                     {{/each}}
-                  </div>
+                  {{#if high_priority}}
```
Comment (russellholt) [inline src/templates/issues.handlebars]: you have full control over the data, so there has to be a better way than asking a question in the template

File: src/js/issues.js [hunk 2]
```diff
@@ -1,11 +1,57 @@
...
+    var result = label.substring(label.indexOf(':') + 1, label.length);
+    return result.trim();
+}
+
+function groupByLanguage(issues) {
+    var by_lang = {};
+    by_lang.languages = langs;
+
+    $.each(issues, function(issue) {
+        $.each(issue.labels, function(label) {
```
Comment (russellholt) [inline src/js/issues.js]: surely something nicer than nested loops with nested if statements can be done about this?

**Comment by MariagraziaAlastra** [comment]
@russellholt Yo, ready for another review, @jagtalon is doing the styling separately

**Comment by russellholt** [comment]
lgtm

**Pull request merged**